packages/@aws-cdk/aws-appmesh/lib/gateway-route-spec.ts

```diff
@@ -29,7 +29,7 @@ export interface GrpcGatewayRouteMatch {
 /**
  * Properties specific for HTTP Based GatewayRoutes
  */
-export interface HttpRouteSpecProps {
+export interface HttpRouteSpecOptions {
```

Comment:
I think there can be a non-Gateway Route Spec as well, right...?

If that's the case, I think these interfaces should be called `HttpGatewayRouteSpecOptions`.

Reply:
Ah yeah good call.